docs/test-prep.md

```diff
@@ -0,0 +1,406 @@
+# Preparing a test for MFTF
+
+This tutorial will guide you through the process of converting a raw functional test into a properly abstracted test file, ready for publishing.
+
+## The abstraction process
```

Comment:
Consider adding a diagram for the abstraction.

Reply:
I think that is not needed here.